analysis/diagnosis/covariates_matrix.R

```diff
@@ -0,0 +1,94 @@
+library(plyr)
+library(dplyr)
+library(tidyr)
+library(readr)
+library(survival)
+
+# Define diagnosis output folder ------------------------------------------
+print("Creating diagnosis output folder")
+
+diag_dir <- "output/diagnosis/"
+fs::dir_create(here::here(diag_dir))
+
+# Load data and interval generator
+source("analysis/diagnosis/fn-generate_time_interval_indicators.R")
+df <- readRDS(
+  "output/model/model_input-cohort_vax-main_preex_FALSE-pneumonia.rds"
```

Comment:
I would make the name an argument in the YAML so the script can be re-used easily.